### DISCUSSION TITLE
Add some google gemini into it ?

### DISCUSSION THREAD

**Discussion by estrizal** [FuseFairy/DiscordBot-EdgeGPT#115] [Ideas]
gemini pro and gemini vision api is free to use as of now. maybe add an option to use it with the bot. Would be really cool, As it responds super fast. 

Though.. it's a bing bot. so, would be a bit weird to add gemini into it too.

**Comment by FuseFairy** [comment]
Maybe will add here, not sure
https://github.com/FuseFairy/DiscordBot-BardGPT

**Reply by estrizal** [reply]
Noiceee~ btw could you make it so that the bot automatically creates the user_chatbot for the user with bot owner cookies, as soon as someone uses /bing or mentions it. 

A lot of the people in the server were not using the bot coz it required then to type "ONE SINGLE '/set_chatbot' COMMAND". Just removed the whole chat_bot thing today, only now they are generating the images etc. 

keep the /set_chatbot command to change the cookies from owner to their own.

**Comment by FuseFairy** [comment]
Okay, now using `/bing`, `/create image`, `/switch style`. `/reset conversation` will automatically create the user chatbot.

I plan to create a threat when a user uses `/bing` . This way everyone goes on their own chat channel.

**Comment by estrizal** [comment]
Perfecto!
